### ISSUE TITLE
Using Gradient on continues rectangle gives an error

### ISSUE THREAD

**Issue by rymesaint** [iampawan/VelocityX#187]
**Describe the bug**
I got an error using gradient when using continuesRectangle()

```dart
'Sign In'
.text
.white
.makeCentered()
.continuousRectangle(
  gradient: const LinearGradient(
      colors: [Colors.black, Colors.white]),
  height: 40,
  width: 120,
)
.onTap(controller.signIn)
```

**Comment by huzaifansari54** [comment]
can u tell what exactly error u have face? use screenshot

**Comment by Shreemanarjun** [comment]
@iampawan I want to  work on it.

**Comment by Shreemanarjun** [comment]
Fixed with PR #189

**Comment by Shreemanarjun** [comment]
### PLEASE LABEL THIS ONE AS HACKTOBERFEST IF POSSIBLE. @iampawan
